dropwizard-metrics-graphite/src/main/java/io/dropwizard/metrics/graphite/GraphiteReporterFactory.java

```diff
@@ -51,6 +58,9 @@
     @NotNull
     private String prefix = "";
 
+    @NotNull
```

Comment:
Yea, didn't know that was available. With that annotation, will drop the enum and just use strings/ignore case